src/libraries/Common/src/System/Security/Cryptography/Asn1/GeneralSubtreeAsn.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<asn:Sequence
+  xmlns:asn="http://schemas.dot.net/asnxml/201808/"
+  name="GeneralSubtreeAsn"
+  namespace="System.Security.Cryptography.X509Certificates.Asn1">
+
+  <!--
+    https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.10
+
+      GeneralSubtree ::= SEQUENCE {
+          base                    GeneralName,
+          minimum         [0]     BaseDistance DEFAULT 0,
+          maximum         [1]     BaseDistance OPTIONAL }
+
+      BaseDistance ::= INTEGER (0..MAX)
+  -->
+  <asn:AsnType name="@base" typeName="System.Security.Cryptography.Asn1.GeneralNameAsn" />
+  <asn:Integer name="minimum" backingType="int" explicitTag="0" defaultDerInit="0x02, 0x01, 0x00" />
+  <asn:Integer name="maximum" backingType="int" explicitTag="1" optional="true" />
```

Comment:
I believe we've (I've) generally used PascalCasing (even though that's wrong for fields... were they ever properties?).  It solves the need for "@base", and makes "Defaultminimum" => "DefaultMinimum"

Reply:
I don't think they've ever been properties.